src/main/java/org/apache/sling/jcr/repoinit/impl/AclVisitor.java

```diff
@@ -142,7 +142,9 @@ public void visitCreatePath(CreatePath cp) {
         for (PathSegmentDefinition psd : cp.getDefinitions()) {
             final String fullPath = parentPath + "/" + psd.getSegment();
             try {
-                if (session.itemExists(fullPath)) {
+                if (session.propertyExists(fullPath)) {
+                    throw new RuntimeException("Error creating Node at " + fullPath + ": a property exists with the same path");
+                } else if (session.itemExists(fullPath)) {
                     log.info("Path already exists, nothing to do (and not checking its primary type for now): {}", fullPath);
```

Comment:
log.info("Node at {} already exists.....
that makes it unambiguous that we are trying to create a node and not a property. "Path already exists" is a bit vague....

Reply:
Agreed, I have made that change and also changed the log message to say "Node"